jkarenzi/task-master-be
# #1 Set up project

**what does this PR do?**
- This PR setups the project by creating project structure, installing necessary dependencies, and handling various configurations

**Description of the task to be completed**
- [x] Set up empty express project, with basic dependencies
- [x] Set up CI/CD
- [x] Set up test environment using jest & code coverage
- [x] Set up swagger documentation for the project
- [x] Set up global error handler
- [x] Write comprehensive README file for project

**How can this be manually tested?**
- By accessing the swagger doc route (api-docs)
- By accessing the api/test route to check if a json response is returned
- By running npm test:ci to run tests, and check if test setup was done correctly

**What are the relevant pivotal trackers/story id?**
- #1
